beams/public/js/job_requisition.js

```diff
@@ -5,30 +5,40 @@ frappe.ui.form.on('Job Requisition', {
             if (frm.doc.request_for === 'Employee Exit') {
                 return {
                     filters: {
-                        status: 'Left'  // Assuming 'status' is the field that indicates the employee's status
-                    }
-                };
-            } else {
-                return {
-                    filters: {
-                        // No filter if it's not Employee Exit
+                        status: 'Left'  // Filter for employees who have left
                     }
                 };
             }
+            return {};  // No filter if it's not Employee Exit
         });
 
-        /*
-         * Sets a filter on the Job Description Template field based on the Designation.
-         * Clears the Job Description Template field when the form is refreshed.
-         */
+        // Rename Actions Button  as 'Create'  not primary action button
+         $(document).ready(function () {
+              // Find the specific custom "Actions" button dropdown using the parent or group
+              var dropdownActions = frm.page.inner_toolbar
+                  .find('button:contains("Actions")')
+                  .first();
+
+              // Rename this specific "Actions" button to "Create"
+              if (dropdownActions.length) {
+                  dropdownActions.text("Create");
+              }
+          });
```

Comment:
Instead of changing the button label, Remove and re-create the button. Else need to rename the inner buttons too, **Create -> Create Job Opening** is not required. It should be **Create -> Job Opening**